### DISCUSSION TITLE
Link to server documents

### DISCUSSION THREAD

**Discussion by ToGabhart** [senshu/Sozi#665] [Q&A]
I'm using Sozi for a little different purpose rather than as a presentation app so, please understand this is a very out of the box question.
Basically, I am attempting to make it a graphically oriented database for info, documents and reference.
What I am seeking to do is to have elements in Sozi that have a link to a server document in order to launch that document for viewing. Primarily, it is pdf's that I want to be able to open from Sozi

Example: Sozi element has embedded link to a file on a connected server. Clicking the element either launches the pdf in a separate browser window/tab or launches it with Adobe Acrobat.

Is this possible? How can it be done.
Currently the pdf's I am referring to do not have a web address and only exist on a server.

**Comment by aumouvantsillage** [comment]
You can always create links to any file, local or remote, as long as you can provide a URL for this file.
Links are a feature of the SVG format and are not specific to Sozi. You can find more on this topic in this section:
https://sozi.baierouge.fr/pages/tutorial-links.html#creating-hyperlinks-in-a-sozi-presentation
